components/dashboard/cells/ImplementationPhaseCell.tsx

```diff
@@ -0,0 +1,30 @@
+import { ActionListAction } from '../dashboard.types';
+import styled from 'styled-components';
+import { PlanContextFragment } from 'common/__generated__/graphql';
+import ActionPhase from 'components/actions/ActionPhase';
+
+interface Props {
+  action: ActionListAction;
+  plan: PlanContextFragment;
+}
+
+const StatusDisplay = styled.div`
+  padding: ${(props) => props.theme.spaces.s050};
+  height: 100%;
+`;
+
+const ImplementationPhaseCell = ({ action, plan }: Props) => (
+  <StatusDisplay>
+    <ActionPhase
+      action={action}
+      status={action.statusSummary}
```

Comment:
Does it makes sense to pass both `action` and then properties from inside `action` separately?